test/framework/gardener_utils.go

```diff
@@ -95,6 +95,15 @@ func (f *GardenerFramework) GetShootProject(ctx context.Context, shootNamespace
 	return project, nil
 }
 
+// CreateShootResource creates a shoot from a shoot Object
+func (f *GardenerFramework) CreateShootResource(ctx context.Context, shoot *gardencorev1beta1.Shoot) (*gardencorev1beta1.Shoot, error) {
```

Comment:
Do we really a wrapper function for this? It does not abstract that much.

Reply:
I can remove it, I just thought it would be helpful to also have a function that solely creates the resource instead of also [waiting for its reconciliation](https://github.com/danielfoehrKn/gardener/blob/b54ffaf603941c189f012b1a3df1443fb37412bc/test/integration/framework/shoot_operations.go#L89). WDYT